macros/cross-adapter-modeling/base/segment_web_page_views.sql

```diff
@@ -0,0 +1,60 @@
+{% macro segment_web_page_views() %}
+
+    {{ adapter_macro('segment.segment_web_page_views') }}
+
+{% endmacro %}
+
+
+{% macro default__segment_web_page_views() %}
+
+{{ exceptions.raise_compiler_error("macro segment_web_page_views not implemented for this adapter") }}
+
+{% endmacro %}
+
+
+{% macro snowflake__segment_web_page_views() %}
+
+with source as (
+
+    select * from {{var('segment_page_views_table')}}
+    
+),
+
+renamed as (
+
+    select
+    
+        id as page_view_id,
+        anonymous_id,
+        user_id,
+        
+        received_at as received_at_tstamp,
+        sent_at as sent_at_tstamp,        
+        timestamp as tstamp,
+
+        url as page_url,
+        parse_url(url)['host']::varchar as page_url_host,
+        path as page_url_path,
+        title as page_title,
+        search as page_url_query,
+        
+        referrer,
+        parse_url(referrer)['host']::varchar as referrer_host,
```

Comment:
Should this have a `nullif(...,'')` as well? i.e. If there is no referrer, is the `host` key missing or does it have value `''`?